src/PackageManagement.UI/PackageLoader.cs

```diff
@@ -226,10 +226,22 @@ private IEnumerable<string> GetSupportedFrameworks()
             var localResource = await _packageManager.PackagesFolderSourceRepository
                 .GetResourceAsync<UIMetadataResource>();
 
-            var metadataResource =
+            // UIMetadataResource may not be available
+            // Given that this is the 'Installed' filter, we ignore failures in reaching the remote server
+            // Instead, we will use the local UIMetadataResource
+            UIMetadataResource metadataResource;
+            try
+            {
+                metadataResource =
                 _sourceRepository == null ?
                 null :
                 await _sourceRepository.GetResourceAsync<UIMetadataResource>();
+            }
+            catch (Exception)
```

Comment:
We should consider wrapping the service index exceptions in a protocol exception to make this easier to catch. This same issue likely exists in many other scenarios.